contracts/libs/0.8.x/BytecodeStorageV1.sol

```diff
@@ -187,20 +218,110 @@ library BytecodeStorage {
         }
     }
 
+    /**
+     * @notice Get version for given contract bytecode
+     * @param _address address of deployed contract with bytecode containing concat(invalid opcode, version, deployer-address, data)
+     * @return version version read from contract bytecode
+     */
+    function getLibraryVersionForBytecode(
+        address _address
+    ) internal view returns (bytes32) {
+        return _bytecodeVersionAt(_address);
+    }
```

Comment:
any reason to not just make this public?

Reply:
This was just internal for now, as all of the readers are internal for now until we make the split to separate public external library for reads and internal library for writes – so done in accordance with that convention until we make the bigger iterative split. That seem right? @ryley-o Happy to adjust if not.

Reply:
that sounds good!